src/screens/hardware-wallets/components/ActionButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Text } from '@/design-system';
+import React from 'react';
+import { ButtonPressAnimation } from '@/components/animations';
+
+type Props = {
+  onPress: () => void;
+  label: string;
+};
+
+export const ActionButton = ({ onPress, label }: Props) => (
+  <Box width="full" paddingHorizontal="20px">
+    <Box
+      as={ButtonPressAnimation}
+      background="purple"
+      borderRadius={99}
+      alignItems="center"
+      justifyContent="center"
+      width="full"
+      height={{ custom: 56 }}
+      // @ts-expect-error js component
```

Comment:
Yeah why does our Box not support onPress

Reply:
had some strong options around layout components being pure and ~only~ for layout in the DS I believe